components/match2/wikis/autochess/get_match_group_copy_paste_wiki.lua

```diff
@@ -34,7 +34,7 @@ function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
 		INDENT .. '|p1=9 |p2=7 |p3=6 |p4=5 |p5=3 |p6=2 |p7=1 |p8=0',
 		{INDENT .. '|twitch=|youtube='},
 		Array.map(Array.range(1, bestof), function(mapIndex)
-			return INDENT .. '|map' .. mapIndex .. '={{Map|date=|finished=|map=|vod=}}'
+			return INDENT .. '|map' .. mapIndex .. '={{Map|date=|finished=|vod=}}'
```

Comment:
intended to be in this PR?

Reply:
yup just something that was caught when doing the TFT one, that the other autobattlers do not have maps as well